components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -1319,6 +1322,9 @@ void ConversationDriver::OnTextEmbedderInitialized(
                        std::move(data_received_callback),
                        std::move(data_completed_callback), page_content,
                        is_video));
+    UpdateOrCreateLastAssistantEntry(
```

Comment:
Do we need to check that the navigation id / conversation is the same? This is fixed in the refactor but maybe we should be passing navigation_id to `OnTextEmbedderInitialized`?

Reply:
check added in 5dd635b0e67b1c79788bbf4871329dea16be4bbb